src/setuptools_scm/version.py

```diff
@@ -188,7 +188,10 @@ def meta(
             "meta invoked without explicit configuration,"
             " will use defaults where required."
         )
-    parsed_version = _parse_tag(tag, preformatted, config)
+    if config.skip_version_parser:
```

Comment:
the issue already outlined a acceptable structural solution, this isn't

a random flag to opt out not just of the normalization of the version , but also all other cleanup is not going to cut it
also integration tests are required for new features

i strongly suggest to write tests and to consider a Version subclass that normalizes less

Reply:
I don't quite get the reference. Can you explain what's wrong with the flag?

Reply:
as implemented and placed it takes out a number of features in surprising ways

in the issue before it was discussed to have a Version subclass that prevents normalization of valid versions and enables broader version usage

Reply:
I don't understand you. This is an optional opt-out from the check. The role of SCM plugin in my pipeline is to pass the version from tagged commit to setuptools. Choosing which tag is suitable for release is done on CI level.

Reply:
Your specific use case is not the main goal of setuptools_scm

I stated a way that would allow you to implement your use case in a way that matches the project

Im not going to take ownership of the stopgap solution you propose

Currently Im very thin spread  on time and can't discus /fight over this

Reply:
Reopening so that other people could weight in their opinion.